addons/account/data/mail_template_data.xml

```diff
@@ -8,45 +8,48 @@
         <record id="email_template_edi_invoice" model="mail.template">
             <field name="name">Invoice: Send by email</field>
             <field name="model_id" ref="account.model_account_move"/>
-            <field name="email_from">${(object.invoice_user_id.email_formatted or user.email_formatted) |safe}</field>
+            <field name="email_from">${(object.invoice_user_id.email_formatted or user.email_formatted) }</field>
             <field name="partner_to">${object.partner_id.id}</field>
             <field name="subject">${object.company_id.name} Invoice (Ref ${object.name or 'n/a'})</field>
             <field name="body_html" type="html">
 <div style="margin: 0px; padding: 0px;">
     <p style="margin: 0px; padding: 0px; font-size: 13px;">
         Dear
-        % if object.partner_id.parent_id:
-            ${object.partner_id.name} (${object.partner_id.parent_id.name}),
-        % else:
-            ${object.partner_id.name},
-        % endif
+        <t t-if="object.partner_id.parent_id">
+            <t t-out="object.partner_id.name or ''">Brandon Freeman</t> (<t t-out="object.partner_id.parent_id.name or ''">Azure Interior</t>),
```

Comment:
For most of the diff...
`FalsyString or ''    ` 
 ` or ''`   is useless in Qweb VS Jinja

`<t t-out="object.partner_id.name or ''">Brandon Freeman</t>`
=>
`<t t-out="object.partner_id.name">Brandon Freeman</t>`

` `


`<t t-out="object.partner_id">Brandon Freeman</t>`  => ResPartner()
!=
`<t t-out="object.partner_id or '' ">Brandon Freeman</t>` => Brandon Freeman

ping @Gorash ... what do you think about this last usecase... strange no ?

Reply:
> what do you think about this last usecase... strange no ?

It somewhat makes sense that the first one wouldn't work, but it doesn't really make sense that the second one would: IIRC qwebpy special cases `None` (because that's normal) and `False` (because Odoo).

But maybe we should relax the check and just fallback to the body for any falsy result?

Reply:
@Gorash told me to use `or ''` if I wanted to prevent qweb to get the content of the tag in case of a `False` or `None` value.
As demonstrated here: https://drive.google.com/file/d/1lComR8Y_H3pkV0hC7UflWIQybpJ6Wo33/view

Reply:
:+1: